examples/resources/hpegl_vmaas_load_balancer_profile/nsx_t_lb_profile.tf

```diff
@@ -1,6 +1,7 @@
 # (C) Copyright 2022 Hewlett Packard Enterprise Development LP
 
 resource "hpegl_vmaas_load_balancer_profile" "tf_lb_profile" {
+  lb_id = data.hpegl_vmaas_load_balancer.lb.id  
```

Comment:
For the  "profile_type = "application-profile", do we need cookie details ? I think the example is not right. for the profileType= Persistance, we would need the right cookie details

Reply:
Same comment on the resource definition. Completely different from UI
![image](https://user-images.githubusercontent.com/78580773/177919234-fe36bfe8-ece0-4c65-b012-53af358acd72.png)